### ISSUE TITLE
Question about Train & Test result in the paper

### ISSUE THREAD

**Issue by simonwey** [rawalkhirodkar/MIPNet#3]
Amazing work! Thanks for releasing the source code. Here is my question:
1) Which of the following is MIPNet(hrnet backbone)'s final result on crowdpose test set in the paper?
    #-----------------------------------------------------------------------------------
    [code] lib/core/validate.py: def validate_lambda_quantitative():
        _print_name_value(name_values, 'l0,1:{}'.format(model_name))
        _print_name_value(name_values_mode0, 'l0:{}'.format(model_name))
        _print_name_value(name_values_mode1, 'l1:{}'.format(model_name))
    #------------------------------------------------------------------------------------
2) Has MIPNet with hrnet backbone used pretrained model? (about result on crowdpose in the paper)

By the way, Very looking forward to the detailed README.

**Comment by rawalkhirodkar** [comment]
Thank you for your interest.

1. l0,1 mode is the final result reported in the paper.
2. Yes, we initialize the MIPNet with HRNet trained on the COCO dataset.

**Comment by simonwey** [comment]
Got it, thanks for your reply.
